config/manager/manager.yaml

```diff
@@ -59,7 +59,8 @@ spec:
         - command:
             - /manager
           args:
-            - --enable-leader-election
+            - --metrics-bind-address=127.0.0.1:8080
+            - --leader-elect
```

Comment:
This looks like this is fixing a bug from cbebd561bfa0a3967db447bb051cc451e4447e65 that turned `enable-leader-election` into `leader-elect`. This was a long long time ago so I am a bit surprised this wasn't discovered before. How did you spot it ? Code reading ? Hit some error ? Also, is `metrics-bind-address` needed here ?

Cc @bpradipt as author of cbebd561bfa0a3967db447bb051cc451e4447e65.

Reply:
When I tested my change, I faced an issue, try to use a non-existing argument. I saw that the it's changed in clusterserviceversion.yaml during bundle creation. It's interesting, because when I build it without these changes (with 1.36.0 operator-sdk), it didn't changed this arguments. 
Maybe, because of removing/adding manifests to clusterserviceversion reload manager manifest as well...

Default, metrics-bind-address is ":8080", so I think it's needed. 
Also this value is used currently: https://github.com/openshift/sandboxed-containers-operator/blob/devel/bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml#L373

Reply:
I wonder how it worked till now with the incorrect option in manager.yaml - https://github.com/openshift/sandboxed-containers-operator/blob/devel/main.go#L97-L98
Only possibility is that the config/manager/manager.yaml never got used.
Anyways, thanks @balintTobik for fixing this.